src/middlewared/middlewared/alert/source/multipaths.py

```diff
@@ -9,8 +9,8 @@ class MultipathsAlertSource(AlertSource):
 
     async def check(self):
         return [
-            Alert(title="Multipath %s is not optimal",
-                  args=[mp.name])
-            for mp in await self.middleware.call("notifier.multipath_all")
-            if mp.status != "OPTIMAL"
+            Alert(title=f"Multipath {mp['name']} is not optimal",
+                  args=[mp['name']])
```

Comment:
If we dont longer have a %s, why is there still a `args`?

Reply:
@sonicaj sorry I've missed it, we should still have %s there